We do not have pytest-instafail packaged yet.
It is not needed to run tests.

--- pyftpdlib-2.1.0/setup.py.orig
+++ pyftpdlib-2.1.0/setup.py
@@ -19,7 +19,6 @@
     "psutil",
     "pyopenssl",
     "pytest",
-    "pytest-instafail",
     "pytest-xdist",
     "setuptools",
 ]
--- pyftpdlib-2.1.0/pyproject.toml.orig
+++ pyftpdlib-2.1.0/pyproject.toml
@@ -136,12 +136,10 @@
     --tb=short
     --strict-config
     --strict-markers
-    --instafail
     -p no:junitxml
     -p no:doctest
     -p no:nose
     -p no:pastebin
-    -p instafail
     -p xdist
     '''
 testpaths = ["tests/"]
